### ISSUE TITLE
Is it possible to add a caption option under tables? 

### ISSUE THREAD

**Issue by jo-faunalytics** [Faunalytics/faunalytics#5]
No worries if not, but it's something we did sometimes in Infogram and it means that the caption format and width match the table rather than the Wordpress body text. :)

**Comment by zach-faunalytics** [comment]
Yes, I can definitely add this, BUT: per Karol's request, the tables will not have a specific font specified so that they inherit the font (Roboto?) from Wordpress. That mean's we'll have to decide how we want the table caption formatted so I can build in pre-formatting for just that part of the table. Any thoughts you have on font, font size, font color would be greatly appreciated :)

**Comment by jo-faunalytics** [comment]
Gotcha gotcha, that makes total sense! Honestly, the only requirement from my perspective is that it's a small font. Otherwise no special colour or anything, and matching the WP font is fine. See tables in this report for examples: https://faunalytics.org/going-veg-motivations-and-influences/ (And having the option of italicizing is nice but not required -- I assume that might work the same way as it does in the table itself, where we have to do it manually in the HTML code.)

**Comment by zach-faunalytics** [comment]
Okay this has been added in (see the `caption` argument), but there are some limitations. Italics have to be added in manually, as you expected. Also, the way note functionality for HTML tables works is basically by storing the the note in a new row of the the table that's one big cell spanning all of the columns. Because of that, if the note text is longer than the table is wide, the table will stretch up to 100% of the container (e.g. the tab on the website) it's in before wrapping in order to accommodate the text. You can go in and add `<br>` to the HTML wherever you want line breaks. Not ideal, but workable.

I'm going to leave this open until I can think of a solution to make it work better. Note to self on this: probably involves a new div or p below the table that somehow inherits the width and padding of the table.

**Comment by jo-faunalytics** [comment]
Thanks for looking into it! This sounds like a great solution even if it's not 100% of what you want. Good enough for me. :)

**Comment by jo-faunalytics** [comment]
I think it's probably part of this issue, but it looks like `\n` doesn't work within these captions. Any way around that, or just the HTML fix? :)
